### ISSUE TITLE
make sure to delete pull-secret.txt from downloads once tearing down the environment

### ISSUE THREAD

**Issue by yerdreic** [coffr/elk#6]

**Comment by yerdreic** [comment]
check if we don't need to pull new pull-secret.txt file everytime we run make up

**Comment by lmilbaum** [comment]
> check if we don't need to pull new pull-secret.txt file everytime we run make up

I would suggest to treat `pull-secret.txt` as a secret file. Store it in a secrets store and consume it dynamically.

**Comment by yerdreic** [comment]
@lmilbaum oh I see I added a comment instead of changing the issue's name.
The issue refers to if this pull-secret.txt file is constant - meaning, we can pull is once, then use it all the time - Or, is this file change, so we need to update it - meaning, we need to pull it again from time to time.

I think I will first try to find out what is the purpose of this file in order to answer it

**Comment by lmilbaum** [comment]
> @lmilbaum oh I see I added a comment instead of changing the issue's name. The issue refers to if this pull-secret.txt file is constant - meaning, we can pull is once, then use it all the time - Or, is this file change, so we need to update it - meaning, we need to pull it again from time to time.
> 
> I think I will first try to find out what is the purpose of this file in order to answer it

Yup. Makes sense.  I think I know the answer but I am not 100% sure. The CRC folks and Tulle can provide the answer.
